hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java

```diff
@@ -425,14 +423,14 @@ ContainerCommandResponseProto createContainer(
     return handler.handle(requestBuilder.build(), null, null);
   }
 
-  private void validateBlockToken(
+  private void validateToken(
       ContainerCommandRequestProto msg) throws IOException {
-    if (isBlockTokenEnabled && tokenVerifier != null &&
-        HddsUtils.requireBlockToken(msg.getCmdType())) {
+
+    if (tokenVerifier != null) {
```

Comment:
Do we still need the null check with the NoopTokenVerifier?

Reply:
Removed.